Sources/Orbit/Components/CountryFlag.swift

```diff
@@ -86,7 +88,8 @@ struct CountryFlagPreviews: PreviewProvider {
     static var standalone: some View {
         VStack {
             CountryFlag("cz")
-            CountryFlag("")     // EmptyView
+            CountryFlag("") // unknown flag
+            CountryFlag("some invalid identifier") // unknown flag
```

Comment:
This would now render the unknown flag, which should go into a separate property (`var unknown`). The original behaviour was to draw nothing (EmptyView), which was the intent of the VStack. Now, the VStack can be removed and the standalone preview should only show the single `CountryFlag("cz")`.

Reply:
👍